rotkehlchen/history/events.py

```diff
@@ -524,6 +559,17 @@ def fail_history_cb(error_msg: str) -> None:
                 to_timestamp=end_ts,
                 addresses=self.chain_manager.queried_addresses_for_module('liquity'),
             ))
+        step = self._increase_progress(step, total_steps)
+
+        # Include base history entries
+        history_events_db = DBHistoryEvents(self.db)
+        base_entries, _ = history_events_db.get_history_events_and_limit_info(
+            filter_query=HistoryEventFilterQuery.make(
+                event_types=[HistoryEventType.STAKING],
```

Comment:
So probably soon this should change to encompass all history events, right? To get in the transactions too.